app/models/post.rb

```diff
@@ -1,17 +1,15 @@
 class Post < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :author, class_name: 'User', foreign_key: 'user_id'
   has_many :comments
   has_many :likes
 
-  def load_recent_five_comments
-    comments.order('created_at Desc').limit(5)
-  end
-
-  after_save :update_posts_counter
-
-  private
+  after_save :update_post_counter
 
   def update_posts_counter
     author.increment!(:posts_counter)
   end
+
+  def load_recent_five_comments
+    comment.order(created_at: :desc).limit(5)
+  end
```

Comment:
- [x] Let's ensure that the `comment` on line 13 is `comments` so that we can get the 5 most recent comments from the `comments` table.

```suggestion
  def load_recent_five_comments
    comments.order(created_at: :desc).limit(5)
  end
```